packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5428,16 +5429,31 @@ export default class Meeting extends StatelessWebexPlugin {
           },
         };
 
-        this.cleanupLocalStreams();
-
         this.mediaProperties.setMediaDirection(mediaSettings.mediaDirection);
         this.mediaProperties.unsetRemoteMedia();
 
         // when a move to is intiated by the client , Locus delets the existing media node from the server as soon the DX answers the meeting
-        // once the DX answers we establish connection back the media server with only receiveShare enabled
-        // @ts-ignore - reconnectMedia does not accept any argument
-        await this.reconnectionManager.reconnectMedia(mediaSettings).then(() => {
+        // once the DX answers we close the old connection and create new media server connection with only share enabled
+        const stopStatsAnalyzer = this.statsAnalyzer
+          ? this.statsAnalyzer.stopAnalyzer()
+          : Promise.resolve();
+        await stopStatsAnalyzer
```

Comment:
this whole block of code from line 5437 to 5448 is now a mix of `await` and `then`. There is no reason for that, let's try to clean it up and just use await for all the calls. We also don't need to do the stuff from lines 5437-5439, we can just simply do this:
```
if (this.statsAnalyzer) {
  await this.statsAnalyzer.stopAnalyzer();
}
```

Reply:
We still have to cleanup the streams right?

Reply:
yes

Reply:
So, I believe the code now is correct. Shall I resolve this conversation?